pennylane/devices/_qubit_device.py

```diff
@@ -758,13 +758,13 @@ def statistics(
             elif isinstance(m, (SampleMeasurement, StateMeasurement)):
                 result = self._measure(m, shot_range=shot_range, bin_size=bin_size)
 
-            elif m.return_type is not None:
+            else:
                 name = obs.name if isinstance(obs, qml.operation.Operator) else type(obs).__name__
                 raise qml.QuantumFunctionError(
                     f"Unsupported return type specified for observable {name}"
                 )
-            else:
-                result = None
+            # else:
+            #     result = None
```

Comment:
Is this meant to be removed?

Reply:
fixed at https://github.com/PennyLaneAI/pennylane/pull/6841/commits/d1df38fd076d27f34673b507af7a7a275221567f